lib/mixpanel-node.d.ts

```diff
@@ -0,0 +1,159 @@
+declare const mixpanel: mixpanel.Mixpanel;
+
+declare namespace mixpanel {
+  export type DistinctId = string;
+
+  export type ErrorCallback = (err: Error | undefined) => any;
```

Comment:
This shouldn't be an ErrorCallback if it can also call incase of success. How about simply just a `Callback` and a `BatchCallback` ?